frontend/src/views/Login.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="login">
+    <Form @submit="submit">
+      <h1>내 지원서 보기</h1>
+      <TextField
+        v-model="name"
+        name="name"
+        type="text"
+        label="이름"
+        placeholder="이름을 입력해 주세요."
+        :rules="rules.name"
+        required
+      />
+      <TextField
+        v-model="email"
+        name="email"
+        type="email"
+        label="이메일"
+        placeholder="이메일 주소를 입력해 주세요."
+        :rules="rules.email"
+        required
+      />
+      <BirthField v-model="birth" required />
+      <TextField
+        v-model="password"
+        name="password"
+        type="password"
+        label="비밀번호"
+        placeholder="비밀번호를 입력해 주세요."
+        :rules="rules.password"
+        required
+      />
+      <div class="actions">
+        <Button type="button" @click="back" cancel value="이전" />
+        <Button type="submit" value="확인" />
+      </div>
+      <footer>
+        <a class="logo" href="#"></a>
+        <Label class="click" @click.native="findPassword">비밀번호 찾기</Label>
+      </footer>
+    </Form>
+  </div>
+</template>
+
+<script>
+import { Form, Button, TextField, BirthField, Label } from "@/components/form"
+import { login } from "@/utils/validation"
+
+export default {
+  name: "Login",
+  components: {
+    Form,
+    Button,
+    TextField,
+    BirthField,
+    Label,
+  },
+  data: () => ({
+    name: "",
+    email: "",
+    password: "",
+    birth: {
+      year: "",
+      month: "",
+      day: "",
```

Comment:
```suggestion
    birth: {
      year: "",
      month: "",
      day: "",
```

#34 이슈에는 생년월일 유효성 체크가 있는데 PR엔 없는 것 같아요

Reply:
오호.. 추가했습니다